targets/lenovo-x1/flake-module.nix

```diff
@@ -3,15 +3,71 @@
 #
 # Configuration for Lenovo X1 Carbon Gen 11
 {
-  inputs,
   lib,
   self,
   ...
 }: let
-  inherit (inputs) microvm lanzaboote disko;
-  name = "lenovo-x1-carbon";
   system = "x86_64-linux";
-  targets = import ./everything.nix {inherit self lib microvm lanzaboote disko name system;};
+
+  mvp-definition = machineType: variant: extraModules: let
```

Comment:
This file seems generic enough now to be a `targets/laptop`? And `mvp-definition` could be `laptop-definition` as mvp settings are now a profile.

Reply:
Glad you agree 😉